static/css/foundation.min.css

```diff
@@ -528,7 +528,7 @@ td {
   padding: 0; }
 
 p {
-  margin-bottom: 1rem;
```

Comment:
Can you make this change in main.css? You should be able to add it to the top of the main.css file and revert this back to defaults. This is to avoid Foundation making any changes to their css file that wipes ours.